data_steward/gcloud/gsm/__init__.py

```diff
@@ -23,3 +23,20 @@ def build_secret_full_name(secret_name: str,
         :return: FQN of secret based on provided values
         """
         return f'projects/{project_id}/secrets/{secret_name}/versions/{secret_version}'
+
+    @staticmethod
+    def get_secret_from_secret_manager(secret_name, project_id):
```

Comment:
For consistency in this module, we should try to use parameter typing here.  

Also, if you allow `project_id` to default the same way it is defaulting in lines 14-15, then we can remove the function definition of `_get_mandrill_secret()` in `email_notification.py` and replace all calls of `_get_mandrill_secret()` with `get_secret_from_secret_manager(consts.MANDRILL_TOKEN_SECRET_ID)`.  The project name would default as expected and your test code should continue to work.